src/util/file.cpp

```diff
@@ -1,5 +1,36 @@
 #include "util/file.h"
 
+#include <QRegularExpression>
+
+namespace {
+
+const QRegularExpression kExtractExtensionRegex(R"(\(\*\.(.*)\)$)");
```

Comment:
The usage of this Regex is not obvious.
Can you add a comment with an example for a match and a counter example?

Reply:
Ah, I have just discovered the explaination below.
Can we move it here?

Reply:
Are the outer () required?

Reply:
Yes, the syntax for raw string literals is `R"(` to open it and `)"` to close them.

Reply:
The outer `()` won't be part of the regex.